src/simulation_lifecycle/utils/linked_list.c

```diff
@@ -0,0 +1,87 @@
+#include "simulation_lifecycle/utils/linked_list.h"
+
+Node_t * create_node(void *p_data, size_t data_size, Node_t *p_prev, Node_t *p_next) {
+    /* Allocate memory for new node */
+    Node_t *node = (Node_t *) malloc(sizeof(Node_t));
+    node->data = malloc(data_size);
+    node->prev = p_prev;
+    node->next = p_next;
+    /* Copy data to newly allocated memory. */
+    int i;
+    for (i = 0; i < data_size; i++)
```

Comment:
Just a quick reminder that we need to use curly brackets even if there is only one statement.

Reply:
I know these are all in the style guide she proposed but I think it'll be difficukt to follow 100%. I know personally I have tons of habits that I picked up over the years and that go against Cristina's conventions...

I'll do what I can but I'm sure you'll find stuff like that in my eventual code too.

Reply:
Following coding convention is worth 5%. But I'm pretty sure that a few mishaps won't hurt us :-) So far, everything seems to match the coding convention provided by Christina. Román did a great job!

Reply:
I agree with Francoise here. It doesn't need to be in every review and missing it here and there is ok. But before submitting at the very end we should all take a couple of hours to meticulously add brackets and such where they are needed if we miss them.